winui/Ribbon/Overview.md

```diff
@@ -21,9 +21,9 @@ The Syncfusion [WinUI Ribbon](https://www.syncfusion.com/winui-controls/ribbon)
 * **RibbonGallery:** A simple and flexible ribbon gallery feature displays a collection of related commands similar to the gallery feature used in Microsoft Office.
 
 
- ![Overview of WinUI SfRibbon control](Overview-images/syncfusion-winui-ribbon.png)
+ ![Overview of WinUI SfRibbon control](0verview-images/winui-ribbon-overview.png)
```

Comment:
mistakenly added zero here in Image Folder Name. Check and correct it. Avoid careless mistakes in future.

Reply:
Removed